opm/simulators/wells/MSWellHelpers.hpp

```diff
@@ -62,8 +68,18 @@ namespace mswellhelpers
                 }
```

Comment:
`invDXDirect` in above throwing message be following the function name to be sensible.

Reply:
Sorry, I do not understand this comment. Can your rephrase it, please.

Reply:
Thanks. resolved.